src/core/DrumSynth.cpp

```diff
@@ -191,46 +185,59 @@ int DrumSynth::GetPrivateProfileString(const char *sec, const char *key, const c
     QByteArray dat = f.readAll().constData();
     is.str(string(dat.constData(), dat.size()));
 
-    while (is.good()) {
-        if (!inSection) {
-            is.ignore( numeric_limits<streamsize>::max(), '[');
+    while (is.good())
```

Comment:
This block below this line could use better cleaning. Try inverting the condition and use `break`